make lint error

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->

**What happened**:
===========> Run golangci to lint source codes
pkg/platform/provider/baremetal/phases/kubelet/kubelet.go:79:8: S1039: unnecessary use of fmt.Sprintf (gosimple)
	cmd = fmt.Sprintf("kubectl completion bash > /etc/bash_completion.d/kubectl")
	      ^
pkg/platform/provider/baremetal/phases/image/image.go:45:11: S1039: unnecessary use of fmt.Sprintf (gosimple)
				cmd = fmt.Sprintf(" docker info | grep Proxy")
				      ^
pkg/logagent/controller/logagent/logagent_controller.go:813:34: S1039: unnecessary use of fmt.Sprintf (gosimple)
				LogCollector.Status.Reason = fmt.Sprintf("Log Collector is not healthy in status check")
				                             ^
pkg/auth/registry/project/storage/group.go:211:4: S1036: unnecessary guard around map access (gosimple)
			if _, ok := groupPolicyMap[subj.ID]; ok {
			^
pkg/auth/registry/project/storage/user.go:216:4: S1036: unnecessary guard around map access (gosimple)
			if _, ok := userPolicyMap[subj.ID]; ok {
			^
pkg/auth/registry/user/storage/project.go:103:5: S1036: unnecessary guard around map access (gosimple)
				if _, ok := managed[prj]; ok {
				^
make[1]: *** [go.lint] Error 1
make: *** [lint] Error 2

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
make lint

**Anything else we need to know?**:

**Environment**:
- TKE version:
- Global or business cluster:
- Kubernetes version (use `kubectl version`):
- Install addons:
- Others:

--- a/pkg/auth/registry/project/storage/group.go
+++ b/pkg/auth/registry/project/storage/group.go
@@ -208,11 +208,7 @@ func getGroupsPolicyMap(policyList *auth.ProjectPolicyBindingList) map[string][]
 	groupPolicyMap := make(map[string][]string)
 	for _, policy := range policyList.Items {
 		for _, subj := range policy.Spec.Groups {
-			if _, ok := groupPolicyMap[subj.ID]; ok {
-				groupPolicyMap[subj.ID] = append(groupPolicyMap[subj.ID], policy.Spec.PolicyID)
-			} else {
-				groupPolicyMap[subj.ID] = []string{policy.Spec.PolicyID}
-			}
+			groupPolicyMap[subj.ID] = append(groupPolicyMap[subj.ID], policy.Spec.PolicyID)
 		}
 	}
 

--- a/pkg/auth/registry/project/storage/user.go
+++ b/pkg/auth/registry/project/storage/user.go
@@ -213,11 +213,7 @@ func getUserPolicyMap(policyList *auth.ProjectPolicyBindingList) map[string][]st
 	userPolicyMap := make(map[string][]string)
 	for _, policy := range policyList.Items {
 		for _, subj := range policy.Spec.Users {
-			if _, ok := userPolicyMap[subj.ID]; ok {
-				userPolicyMap[subj.ID] = append(userPolicyMap[subj.ID], policy.Spec.PolicyID)
-			} else {
-				userPolicyMap[subj.ID] = []string{policy.Spec.PolicyID}
-			}
+			userPolicyMap[subj.ID] = append(userPolicyMap[subj.ID], policy.Spec.PolicyID)
 		}
 	}
 

--- a/pkg/auth/registry/user/storage/project.go
+++ b/pkg/auth/registry/user/storage/project.go
@@ -100,18 +100,9 @@ func (r *ProjectREST) List(ctx context.Context, options *metainternalversion.Lis
 
 		if strings.HasPrefix(prj, "prj-") {
 			if role == projectOwner {
-				if _, ok := managed[prj]; ok {
-					managed[prj] = append(managed[prj], role)
-				} else {
-					managed[prj] = auth.ExtraValue{role}
-				}
+				managed[prj] = append(managed[prj], role)
 			}
-
-			if _, ok := memberd[prj]; ok {
-				memberd[prj] = append(memberd[prj], role)
-			} else {
-				memberd[prj] = auth.ExtraValue{role}
-			}
+			memberd[prj] = append(memberd[prj], role)
 		}
 	}
 

--- a/pkg/logagent/controller/logagent/logagent_controller.go
+++ b/pkg/logagent/controller/logagent/logagent_controller.go
@@ -810,7 +810,7 @@ func (c *Controller) checkLogCollectorStatus(
 				}
 				LogCollector = LogCollector.DeepCopy()
 				LogCollector.Status.Phase = v1.AddonPhaseChecking
-				LogCollector.Status.Reason = fmt.Sprintf("Log Collector is not healthy in status check")
+				LogCollector.Status.Reason = "Log Collector is not healthy in status check"
 				if err = c.persistUpdate(ctx, LogCollector); err != nil {
 					return false, err
 				}

--- a/pkg/monitor/controller/prometheus/yamls.go
+++ b/pkg/monitor/controller/prometheus/yamls.go
@@ -448,7 +448,7 @@ func scrapeConfigForPrometheus() string {
 }
 
 func recordRulesForPrometheus() string {
-	rules := fmt.Sprintf(`
+	rules := `
 groups:
 - name: k8s-ag-data
   rules:
@@ -1175,7 +1175,7 @@ groups:
 
   - record: k8s_component_etcd_version
     expr: label_replace(max(etcd_server_version) by (server_version),"gitVersion", "$1", "server_version", "(.*)")
-`)
+`
 
 	return rules
 }

--- a/pkg/platform/controller/addon/helm/helm_prober.go
+++ b/pkg/platform/controller/addon/helm/helm_prober.go
@@ -125,7 +125,7 @@ func (p *prober) periodHealthCheck(ctx context.Context, phase v1.AddonPhase, dur
 		p.fetchSyncMap(phase).Range(
 			func(k, v interface{}) bool {
 				key := k.(string)
-				log.Info(fmt.Sprintf("Probe check"), log.String("helm", key), log.String("phase", string(phase)))
+				log.Info("Probe check", log.String("helm", key), log.String("phase", string(phase)))
 				curPhase, err := p.checkAddonHealthz(ctx, key, phase)
 				if curPhase == "" || (curPhase != phase && err == nil) {
 					p.fetchSyncMap(phase).Delete(key)

--- a/pkg/platform/controller/addon/logcollector/controller.go
+++ b/pkg/platform/controller/addon/logcollector/controller.go
@@ -780,7 +780,7 @@ func (c *Controller) checkLogCollectorStatus(
 			if time.Now().After(initDelay) {
 				LogCollector = LogCollector.DeepCopy()
 				LogCollector.Status.Phase = v1.AddonPhaseFailed
-				LogCollector.Status.Reason = fmt.Sprintf("Log Collector is not healthy")
+				LogCollector.Status.Reason = "Log Collector is not healthy"
 				if err = c.persistUpdate(ctx, LogCollector); err != nil {
 					return false, err
 				}

--- a/pkg/platform/controller/addon/prometheus/yamls.go
+++ b/pkg/platform/controller/addon/prometheus/yamls.go
@@ -448,7 +448,7 @@ func scrapeConfigForPrometheus() string {
 }
 
 func recordRulesForPrometheus() string {
-	rules := fmt.Sprintf(`
+	rules := `
 groups:
 - name: k8s-ag-data
   rules:
@@ -1175,7 +1175,7 @@ groups:
 
   - record: k8s_component_etcd_version
     expr: label_replace(max(etcd_server_version) by (server_version),"gitVersion", "$1", "server_version", "(.*)")
-`)
+`
 
 	return rules
 }

--- a/pkg/platform/provider/baremetal/phases/image/image.go
+++ b/pkg/platform/provider/baremetal/phases/image/image.go
@@ -42,7 +42,7 @@ func PullKubernetesImages(s ssh.Interface, option *Option) error {
 		_, err := s.CombinedOutput(cmd)
 		if err != nil {
 			if strings.Contains(err.Error(), "502 Bad Gateway") {
-				cmd = fmt.Sprintf(" docker info | grep Proxy")
+				cmd = " docker info | grep Proxy"
 				output, _ := s.CombinedOutput(cmd)
 				return fmt.Errorf(`pull image fail: %s. maybe set no_proxy for registry(%s,*.%s) in docker dameon.
 					docker info:%s. see: https://docs.docker.com/config/daemon/systemd/#httphttps-proxy`,

--- a/pkg/platform/provider/baremetal/phases/kubelet/kubelet.go
+++ b/pkg/platform/provider/baremetal/phases/kubelet/kubelet.go
@@ -76,7 +76,7 @@ func Install(s ssh.Interface, version string) (err error) {
 		return err
 	}
 
-	cmd = fmt.Sprintf("kubectl completion bash > /etc/bash_completion.d/kubectl")
+	cmd = "kubectl completion bash > /etc/bash_completion.d/kubectl"
 	_, err = s.CombinedOutput(cmd)
 	if err != nil {
 		return err

